src/app/candidate-form/candidate-form.component.html

```diff
@@ -0,0 +1,61 @@
+{{diagnostic}}
+<div class="container">
+    <h1>Resume Builder Form</h1>
+    <br>
+    <form (ngSubmit)="onSubmit()" #candidateForm="ngForm">
+      <div class="form-group">
+        <label for="name">Name</label>
+        <input type="text" class="form-control" id="name" required [(ngModel)]="model.name" name="name">
+      </div>
+
+      <div class="form-group">
```

Comment:
Nice job on the form! Maybe you could try think of a way to iterate through the questions that you want to make into input fields since most of the fields look very similar.